Frends.AzureBlobStorage.ReadBlob/Frends.AzureBlobStorage.ReadBlob.Tests/UnitTests.cs

```diff
@@ -116,7 +144,7 @@ public void ReadBlobSasMissing()
     [Test]
```

Comment:
https://github.com/FrendsPlatform/Frends.AzureBlobStorage/blob/d5c3e966997aaab86e2c7420dade5ee6ee725b64/Frends.AzureBlobStorage.ReadBlob/Frends.AzureBlobStorage.ReadBlob.Tests/UnitTests.cs#L138
Please use `IsTrue` instead of `That`.
https://github.com/FrendsPlatform/Frends.AzureBlobStorage/blob/d5c3e966997aaab86e2c7420dade5ee6ee725b64/Frends.AzureBlobStorage.ReadBlob/Frends.AzureBlobStorage.ReadBlob.Tests/UnitTests.cs#L160
Same here.